### PR TITLE
Integration1 my recipes

### PR DESCRIPTION
**Pull request by nala-he**
Implemented my recipes component. Screenshots were sent to Teams.

### CODE REVIEW

File: src/components/profile/my-recipes-item.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                        </div>
+                        <div className="btn d-none d-xl-block"
+                             onClick={() => deleteRecipeHandler(recipe._id)}>
+                            <i className="bi bi-trash text-dark p-2"></i>
+                            <span className="wd-text-md">Delete Recipe</span>
+                        </div>
+                    </div>
+                    <div className="col-5 d-flex justify-content-end">
+                        <div className="btn d-block d-xl-none">
+                            <Link to="./details" className="wd-no-decor">
```
Comment (MaceeQi) [inline src/components/profile/my-recipes-item.js]: Can you change the path for the link to include the recipe ID? So we know which recipe details we are opening

**Review by MaceeQi — COMMENTED** [review]
Left a comment on the screenshots on Teams and reiterated one of my comments regarding the path here in the code. Otherwise, looks good : )

**Comment by t1yena** [comment]
i ran it locally and everything looks good! Once we change the path as macee mentioned we can merge it!

**Comment by nala-he** [comment]
I revised the font size and the path as Macee suggested. Screenshots sent to Teams.

**Pull request merged**